script/build.ts

```diff
@@ -83,7 +83,7 @@ async function writeModeOutput(collection: ModeCollection): Promise<void> {
 
   writeTsTypeIndex(collection)
 
-  writeDeprecations(collection)
+  // writeDeprecations(collection)
```

Comment:
We still need this function to parse the `deprecations.json` file
```suggestion
  writeDeprecations(collection)
```

Reply:
On line 172 of this file, we assert that every deprecated variable in `deprecations.json` exists in Primer Primitives. If we want to keep deprecated variables in that file after they're removed, we'll need to remove that assertion.